cmd/goal/clerk.go

```diff
@@ -1229,9 +1229,6 @@ var dryrunRemoteCmd = &cobra.Command{
 						trace = *txnResult.LogicSigTrace
 					}
 				}
-				if txnResult.Cost != nil {
-					fmt.Fprintf(os.Stdout, "tx[%d] cost: %d\n", i, *txnResult.Cost)
```

Comment:
I assumed we also wanted to remove this from the goal outputs - otherwise, we can emit the cost by getting the difference between the consumed and added since the cost field does not exist in the `txnResult` anymore.